analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/NullPointerDereference.CSharp6.cs

```diff
@@ -13,7 +13,8 @@ void ConditionalThisFieldAccess()
         {
             object o = null;
             this._foo1 = o;
-            this?._foo1.ToString(); // FIXME Non-compliant
+            this?._foo1.ToString();
+            //   ^^^^^^              {{'._foo1' is null on at least one execution path.}}
```

Comment:
Notice the `.` in the message. It is produced by the `instance.Syntax.ToString()`